integration_tests/src/test/resources/cases/dump_shards_android.yml

```diff
@@ -0,0 +1,3 @@
+gcloud:
+  app: ../test_runner/src/test/kotlin/ftl/fixtures/tmp/apk/app-debug.apk
+  test: ../test_runner/src/test/kotlin/ftl/fixtures/tmp/apk/app-single-success-debug-androidTest.apk
```

Comment:
Let's switch to multiple success apk, it contains ignored and parameterized classes. The test should also verify if ignored tests are placed in the 'ignored section' in JSON